From: Alexis Murzeau <amubtdx@outlook.fr>
Date: Sat, 7 Oct 2017 19:50:46 +0200
Subject: Fix sphinx warning on Directive class

Sphinx has deprecated sphinx.util.compat.Directive.
Use the class in docutils as recommended by Sphinx.

Forwarded: https://github.com/streamlink/streamlink/pull/1260
Author: Alexis Murzeau <amubtdx@outlook.fr>
---
 docs/ext_argparse.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/docs/ext_argparse.py
+++ b/docs/ext_argparse.py
@@ -17,10 +17,10 @@
 from textwrap import dedent
 
 from docutils import nodes
+from docutils.parsers.rst import Directive
 from docutils.parsers.rst.directives import unchanged
 from docutils.statemachine import ViewList
 from sphinx.util.nodes import nested_parse_with_titles
-from sphinx.util.compat import Directive
 
 
 _ArgumentParser = argparse.ArgumentParser
